src/scripts/main.js

```diff
@@ -1,13 +1,26 @@
 'use strict';
 
-function waitFor(element, eventName) {
-// write your code here
-}
-
 const printMessage = (message) => {
-// write your code here
+  const box1 = document.createElement('div');
```

Comment:
Why do you add numbers to the variable names? Try to avoid it, pease
```suggestion
  const box = document.createElement('div');
```